content/rs/release-notes/rs-6-2-4-august-2021.md

```diff
@@ -0,0 +1,159 @@
+
+- Setting the cluster policy to enable data plane internode encryption by default for new databases
+    
+    ``` shell
+    rladmin tune cluster data_internode_encryption
+    ```
+         
+- Enabling it for individual existing databases
+
+    ``` shell
+    rladmin tune db <db:id | name> data_internode_encryption enabled
+    ```
+
+### Internal certificate management
+
+Internode encryption relies on internal certificates signed by a unique, private CA certificate created for your deployment.  The private CA generates and signs leaf certificates for internode encryption only.  It's generated when you install or upgrade to Redis Enterprise 6.2.4.  it's used only within the cluster and is not exposed outside of the cluster.  
+
+The leaf certificates expire regularly; they're automatically rotated before expiration and alerts are issued as needed.
+
+### Open source Redis compatibility
+
+[Redis 6.2](https://raw.githubusercontent.com/redis/redis/6.2/00-RELEASENOTES) introduced new commands, feature improvements, security fixes; it addresses many customer requests.
+
+Redis Enterprise Software supports all new commands, except [RESET](https://redis.io/commands/reset) and [FAILOVER](https://redis.io/commands/failover).  (Redis Enterprise takes a different approach to connectivity; it also separates control plane operations from data plane operations.)
+
+To learn more, see Redis Enterprise Software [compatibility with open source]({{<relref "/rs/concepts/compatibility.md">}}).
+
+### Redis modules
+
+Redis Enterprise Software v6.2.4 includes the following Redis modules:
+
+- [RediSearch v2.0.11]({{<relref "/modules/redisearch/release-notes/redisearch-2.0-release-notes.md">}})
+- [RedisJSON v1.0.8]({{<relref "/modules/redisjson/release-notes/redisjson-1.0-release-notes.md">}})
+- [RedisBloom v2.2.6]({{<relref "/modules/redisbloom/release-notes/redisbloom-2.2-release-notes.md">}}) 
+- [RedisGraph v2.4.7]({{<relref "/modules/redisgraph/release-notes/redisgraph-2.4-release-notes.md">}})
+- [RedisTimeSeries v1.4.10]({{<relref "/modules/redistimeseries/release-notes/redistimeseries-1.4-release-notes.md">}})
+
+### Internode encryption for modules 
+         
+To utilize data plane encryption for existing databases with modules, update the module to the latest version prior to enabling data plane encryption. 
+
+For help, see [Upgrade the module for a database]({{<relref "//modules/add-module-to-cluster.md#upgrading-the-module-for-the-database">}}).
+
+### Module-related enhancements
+
+Added the capability to update current module arguments for an existing database. 
+In earlier versions, you could do this only when upgrading a module.
+To learn more, see [rladmin upgrade]({{<relref "/rs/references/rladmin.md#upgrade">}}).
+
+## Resolved issues
+
+- RS39954 - Changed the UI status indication for the [default user]({{<relref "/rs/references/rladmin.md#upgrade">}}) from `Active/Inactive` to `Enabled/Disabled` 
```

Comment:
```suggestion
- RS39954 - Changed the UI status indication for the [default user]({{<relref "/rs/references/rladmin#upgrade">}}) from `Active/Inactive` to `Enabled/Disabled` 
```

Reply:
I've made the update, the link still doesn't go to the correct section.  We can ship this update and continue to iterate, as I think there's a glitch in link handling.